tests/cypress/tests/e2e/admin-dashboard.spec.js

```diff
@@ -0,0 +1,59 @@
+// element selectors
+const menuButton = '[data-testid="header-menu-dropdown-button"]';
+const menuOptionLogOut = '[data-testid="header-menu-option-log-out"]';
+const statesDropdown = '[data-testid="admin-dropdown"]';
+const enterProgramButton = '[data-testid="enter-program"]';
```

Comment:
Suggestion: We could change these to use selectors with higher priority in the [selectors best practices list](https://testing-library.com/docs/queries/about/#priority). For instance:

```
cy.findByRole("button", { name: "My Account" }) // menu button
cy.findByRole("button", { name: "Log Out" }) // log out button
cy.get('input[name="state"]') // states dropdown
cy.findByRole("button", { name: "Enter" }) // enter program button
```